### PR TITLE
Update docs to fix ssh issues on new merge day instance.

### PR DESCRIPTION
**Pull request by mozbhearsum**
This clarifies how people should be connecting, and instructs them to set their username when pushing, to avoid the need to do manual ssh config hacks. It depends on https://phabricator.services.mozilla.com/D2228 to work.

### CODE REVIEW

File: docs/mergeduty/merge-and-staging-instance.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 There is an AWS instance to run staging and merging instances so that we are fewer hops away from the hg repos. 
 
-To access it, start the "mergeday1" instance in us-west-2 from the AWS console. You should be able to access it by sshing to mergeday1.srv.releng.usw2.mozilla.com. Be sure to "sudo su - buildduty" before doing any work.
+To access it, start the "mergeday1" instance in us-west-2 from the AWS console. You should be able to access it with:
+ ssh -A buildduty@mergeday1.srv.releng.usw2.mozilla.com
+
+It's important that you connect with agent forwarding enabled, and directly as buildduty to ensure that ssh connections to hg.mozilla.org will work.
```
Comment (MihaiTabara) [inline docs/mergeduty/merge-and-staging-instance.md]: ++++

File: docs/mergeduty/howto.md [hunk 2]
```diff
@@ -122,9 +122,12 @@ There are ftl check hooks on [hg.m.o](http://hg.mozilla.org/) that prevent users
 1. The diff for `release` should be fairly similar to [this](https://hg.mozilla.org/releases/mozilla-release/rev/70e32e6bf15e), with updated branding as well as the version change.
 1. Push your changes generated by no-op trial run:
 ```sh
+# Set this variable to your ldap e-mail address to make sure
```
Comment (MihaiTabara) [inline docs/mergeduty/howto.md]: I think we need to add these instructions at lines 89, 96 and 103 too.

File: docs/mergeduty/merge-and-staging-instance.md [hunk 3]
```diff
@@ -2,4 +2,7 @@
 
 There is an AWS instance to run staging and merging instances so that we are fewer hops away from the hg repos. 
 
-To access it, start the "mergeday1" instance in us-west-2 from the AWS console. You should be able to access it by sshing to mergeday1.srv.releng.usw2.mozilla.com. Be sure to "sudo su - buildduty" before doing any work.
+To access it, start the "mergeday1" instance in us-west-2 from the AWS console. You should be able to access it with:
+ ssh -A buildduty@mergeday1.srv.releng.usw2.mozilla.com
```
[the remark this answers is not in the document]
Comment (mozbhearsum) [reply docs/mergeduty/merge-and-staging-instance.md]: Sounds good, thanks for the verbiage.

File: docs/mergeduty/howto.md (hunk 2 above)
Comment (mozbhearsum) [reply docs/mergeduty/howto.md]: Will do.

**Pull request merged**